checks/raw/packaging_test.go

```diff
@@ -0,0 +1,91 @@
+// Copyright 2021 Security Scorecard Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package raw
+
+import (
+	"context"
+	"testing"
+
+	"github.com/golang/mock/gomock"
+	"github.com/google/go-cmp/cmp"
+	"github.com/google/go-cmp/cmp/cmpopts"
+
+	"github.com/ossf/scorecard/v4/checker"
+	mockrepo "github.com/ossf/scorecard/v4/clients/mockclients"
+	scut "github.com/ossf/scorecard/v4/utests"
+)
+
+func Test_Packaging(t *testing.T) {
+	t.Parallel()
+
+	tests := []struct {
+		name      string
+		filenames []string
+		err       error
+		score     int
```

Comment:
score seems weird to include here in `raw`. Maybe switch this to `want *string` or something? Can be done as part of #2501

Reply:
include the score you mean?

Reply:
I updated the tests to check for the ecosystem name only.